toolkit/tools/pkg/imagecustomizerlib/customizeverity_test.go

```diff
@@ -86,81 +83,6 @@ func testCustomizeImageVerityHelper(t *testing.T, testName string, imageType bas
 		"PARTUUID="+partitions[4].PartUuid, "root")
 }
 
-func TestCustomizeImageVerityShrinkExtract(t *testing.T) {
```

Comment:
All these tests that you have removed are still valid tests. But they need to be updated to use COSI.